utils/hct/hctdb_instrhelp.py

```diff
@@ -1941,7 +1941,6 @@ def RunCodeTagUpdate(file_path):
             "lib/DXIL/DxilShaderModel.cpp",
             "include/dxc/DXIL/DxilConstants.h",
             "include/dxc/DXIL/DxilShaderModel.h",
-            "include/dxc/HLSL/DxilValidation.h",
```

Comment:
This doesn't need to be updated to look at the new location?

Reply:
There's no <py::lines left in DxilValidation.h after https://github.com/microsoft/DirectXShaderCompiler/commit/6cf0a6f7df67c72835d88659d410a43990307fc3